install_config/install/advanced_install.adoc

```diff
@@ -928,6 +932,20 @@ region endpoint parameter:
 openshift_hosted_registry_storage_s3_regionendpoint=https://myendpoint.example.com/
 ----
 
+[discrete]
+[[advanced-install-registry-storage-gce]]
+===== Option E: Google Cloud Storage (GCS) bucket on Google Compute Engine (GCE)
+
+A GCS bucket must already exist.
```

Comment:
True, I had similar thoughts, while working on this. There are other places as well where it would help to link them to their provider docs or move them altogether. Maybe I will take this up as a separate issue in itself. Also,  
1. The other options on page https://docs.openshift.com/container-platform/3.9/install_config/install/advanced_install.html#advanced-install-registry are similar.
2. We have individual provider configuration docs listing requirements, for example, https://docs.openshift.com/container-platform/3.9/install_config/configuring_gce.html